src/rulesets/oas/index.json

```diff
@@ -643,7 +644,7 @@
     },
     "oas3-unused-components-schema": {
```

Comment:
I'd say we can drop the rule at this point.
`oas3-unused-component` somewhat overlaps with it.

Reply:
Dropping the rule entirely might cause unexpected results for users with a configuration that starts by disabling all rules and then selectively enables rules by name.  Since Spectral does not issue a warning about rules it does not recognize in the configuration file, Spectral will silently stop linting their unused schemas.

But I'm happy to remove the rule if you think this is acceptable.

Reply:
Yeah, I'd say let's do it, but curious to hear what @philsturgeon thinks about the idea.

>  Since Spectral does not issue a warning about rules it does not recognize in the configuration file, Spectral will silently stop linting their unused schemas.

We're thinking to change it in the next major version. It's going to throw an error. Failing silently is usually not intended.

Reply:
The next version should be a major release so the impact should not be too unexpected, we'll just have to communicate it clearly.